packages/primitives/button/test/index.ts

```diff
@@ -0,0 +1,44 @@
+import test from 'tape'
+import { renderApp } from '../../shared/test-utils'
+
+type TFixtures = 'bare-bones' | 'disabled'
+const renderButton = (filename: TFixtures) => renderApp(require.resolve(`./fixtures/${filename}.tsx`))
+// const a11yReport = (filename: TFixtures) => getA11yReport(require.resolve(`./fixtures/${filename}.tsx`))
+
+const PACKAGE = '@primitives/button'
+
+test(`${PACKAGE} has proper role`, async (t) => {
+  const page = await renderButton('bare-bones')
+  const btnEl = await page.$('button')
+  const a11ySnapBtn = await page.accessibility.snapshot({ interestingOnly: false, root: btnEl })
+  // Waiting for a fix on r11y side so we can refactor this test
+  // https://github.com/nextools/metarepo/issues/233
+  // const report = await a11yReport('bare-bones')
+
+  t.equal(a11ySnapBtn.role, 'button', 'button role is `button`')
```

Comment:
this is the only test we return a message, WTYT? should we always return a message, should we never return one?

Reply:
imho, always

I was unaware `t.ok()` also allows to pass a message, after checking the docs, I'll add now.

thanks!